### PR TITLE
Plasma profile: Avoid circular deps

### PR DESCRIPTION
**Pull request by a17r**
@gentoo/kde

### CODE REVIEW

File: kde-frameworks/kapidox/kapidox-5.21.0.ebuild [hunk 1]
```diff
@@ -4,16 +4,16 @@
 
 EAPI=6
 
-KDE_DEBUG="false"
-KDE_DOXYGEN="false"
-KDE_TEST="false"
 DISTUTILS_SINGLE_IMPL="true"
 PYTHON_COMPAT=( python2_7 )
-inherit kde5 distutils-r1
```
Comment (johu) [inline kde-frameworks/kapidox/kapidox-5.21.0.ebuild]: We could just use KDE_AUTODEPS for this purpose.

Comment (a17r) [inline kde-frameworks/kapidox/kapidox-5.21.0.ebuild]: possible and i did that at first, but that would still keep the cmake dependency. not sure what purpose kde5.eclass has when it takes more lines to disable stuff than it saves in providing metadata. ;)

Comment (johu) [inline kde-frameworks/kapidox/kapidox-5.21.0.ebuild]: https://qa-reports.gentoo.org/output/genrdeps/dindex/kde-frameworks/kapidox All consumers need cmake. What is the purpose to drop the dependency?

Comment (a17r) [inline kde-frameworks/kapidox/kapidox-5.21.0.ebuild]: Sure, good point, none other than to be super-correct ;)

Comment (johu) [inline kde-frameworks/kapidox/kapidox-5.21.0.ebuild]: Super correct means we loose the uniform handling of kde-frameworks packages. ;)

**Comment by johu** [comment]
Thanks, picked as d90f128734249ca6db01f3701cda552aefa5ecc2 and 30d389b3f075bca89d2ba5dfbd68eefa56d7d189.

**Pull request closed without merging**